### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by TomasMerva** [shakti365/soft-actor-critic#3]
How good is this implementation in terms of performance? Is it close to original? What is the hardest env that this implementation has solved?
Thank you in advance

**Comment by shakti365** [comment]
Hey @TomasMerva, this implementation is not complete. I have only started debugging it on the `MountainCarContinuous-v0` environment. I am still planning to take out time and reproduce the results from original paper but it will take some time. Thanks for asking!

**Comment by zscsjtu** [comment]
Hi, I want to know why the performance is so poor in the MountainCarContinuous-v0? I notice that the train_loss becomes larger and larger and terminates at "nan". And the car does not reach the flag in the end. Is there something wrong?
Thank you~

**Comment by shakti365** [comment]
Hey @zscsjtu, you are right that this implementation doesn't solve the environment. At the time when I was focusing on this project I had only started debugging it for `MountainCarContinuous-v0`. In the process, I realised that I needed better understanding of basic algorithm implementations before I could fix my SAC implementation. This is the reason why I haven't focused a lot of my time on this implementation yet and hence the performance issues.

**Comment by chensaics** [comment]
so, the code does not totally implement the algorithm in original paper, right? if I want to improve the code of yours in my project, do you have some recommendations? your code has many explanations, that is nice.

**Comment by shakti365** [comment]
@chensaics yeah it doesn't totally implement the algorithm, as of now. A good starting point would be to start with a simpler objective function. SAC has 3 different types of objective, I would recommend you start with 1 objective, debug it and then add more objectives. If you would like to collaborate on this we can figure out a plan together. This might just motivate me further to complete the implementation. :sweat_smile:
